### ISSUE TITLE
Dropout module conflicting with that of nn module's 

### ISSUE THREAD

**Issue by kmul00** [Element-Research/rnn#190]
Hi,

Since both `nn` and `rnn` package has a ***Dropout*** module now, performing
```lua
nn.Dropout()
```
calls the one *required* later, that is it calls the `rnn` module's (since I did `require 'rnn'` after doing `require 'nn'`).

Now will that be a problem if I want to use **dropout** both in my **GRU** as well as between two **linear** layers (say) ?

From the code in both `nn` and `rnn`, it appears that it won't be any problem, still just wanted to make sure.

Also if someone does `require 'nn'` later, it will override the call to the `rnn` *dropout*, and then things like *lazy dropout* won't work.

**Comment by kmul00** [comment]
I am sorry. `rnn` module's `Dropout` overrides the `nn` module's counterpart completely.

My apologies for opening an issue before inspecting thoroughly.

**Comment by nicholas-leonard** [comment]
@kmul00 Hey no problem :)
